website/package.json

```
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && next export -o hosting/files/",
```

```suggestion
    "build": "next build && next export -o website/dist/",
```

Keep things standardized?

---

This needs to follow Realm's application configuration. That way when Realm deploys it knows where to look for the bundled files.